tests/end_to_end/conftest.py

```diff
@@ -7,16 +7,22 @@
 from tempfile import gettempdir
 
 from wazuh_testing.tools.file import remove_file
+from wazuh_testing import end_to_end as e2e
 
 
 alerts_json = os.path.join(gettempdir(), 'alerts.json')
 
 
 @pytest.fixture(scope='function')
-def clean_environment():
-    """Delete alerts and credentials files from the temporary folder."""
+def clean_environment(get_dashboard_credentials):
+    """Delete alerts and delete indices using the API.
+
+      Args:
+          credentials(dict): wazuh-indexer credentials.
+    """
     yield
     remove_file(alerts_json)
+    e2e.delete_index_api(credentials=get_dashboard_credentials)
```

Comment:
How do you pass the IP address?

Reply:
@jmv74211 The IP or hostname, if omitted, is defined by default in the `delete_index_api` function as 'wazuh-manager'. This is because we use that hostname as standard in the E2E suite, specifically in playbooks.